client/src/components/shared/header/Header.js

```diff
@@ -29,9 +23,7 @@ export default function Header() {
       position: 'absolute',
       paddingLeft: theme.spacing(5),
       paddingRight: theme.spacing(5),
-      // Here we use our function headerBackgroundColor() to get
-      // our header background color based off of the current route (location.path).
-      background: headerBackgroundColor(route)
+      background: getHeaderColor()
```

Comment:
By changing the method name to describe itself, we need not provide documentation on what this line does.